### PR TITLE
closes #12 This adds a function to get account summaries for both user and admin accounts

### LINKED ISSUE #12: connects https://github.com/AdharaProjects/cash-tokenizer-ui/issues/61 Add `/cbs/accountSummary` endpoint from cbs-proxy to cbs-proxy-client

### PR DESCRIPTION
**Pull request by JasoonS**
This includes the changes for the #22 pr for the cbs-proxy and will only work when running against that version.

Also this has not been published to NPM - we must publish it as soon as the PR is done.

To test:

1. Start cyclos on localhost 4000 (use tag `TAG=cash_tokenizer_ci_1 docker-compose up -d`)
2. run `npm start` for the cbs-proxy (must be latest version)
3. run `npm test` (in the cbs-proxy-client folder).

Also - 
Run `circleci build` ( or `docker stop $(docker ps -a -q); docker rm $(docker ps -a -q); docker network rm cyclos-cbs-proxy-client-test; circleci build` to make sure you have a clean setup)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -49,55 +54,66 @@ const createTokenProxy = async (username, password, url) => {
...
   return {
-    getTransfersToOmnibusAccount,
-    getTransfersFromOmnibusAccount,
-    makeTransferToOmnibusAccount,
-    makeTransferFromOmnibusAccount
+    getTransfersToPrimaryAccount,
+    getTransfersFromPrimaryAccount,
+    makeTransferToAdminPrimaryAccount,
+    makeTransferFromAdminPrimaryAccount,
+    getAccountDetails
```
Comment (coeniebeyers) [inline lib/index.js]: @JasoonS 
v0.0.5 should rather return:
```
{
  getTransfersToPrimaryAccount,
  getTransfersFromPrimaryAccount,
  makeTransferToAdminPrimaryAccount,
  makeTransferFromAdminPrimaryAccount
  getTransfersToOmnibusAccount: getTransfersToPrimaryAccount,
  getTransfersFromOmnibusAccount: getTransfersFromPrimaryAccount,
  makeTransferToOmnibusAccount: makeTransferToAdminPrimaryAccount,
  makeTransferFromOmnibusAccount: makeTransferFromAdminPrimaryAccount,
  getAccountDetails
}
```
or am I missing something?

v0.0.6 can then remove support for the older methods

Comment (JasoonS) [reply lib/index.js]: I don't see too much value in doing it like this at the moment, unless the eta for Rynos work is far in the future. Maybe we can quickly discuss this at standup?

Comment (coeniebeyers) [reply lib/index.js]: I'd prefer to not have to add these renames to his PR, and this allows us to test against the latest cbs-proxy code

File: lib/index.js [hunk 2]
```diff
@@ -112,7 +112,11 @@ const createTokenProxy = async (username, password, url) => {
     getTransfersToPrimaryAccount,
     getTransfersFromPrimaryAccount,
     makeTransferToAdminPrimaryAccount,
-    makeTransferFromAdminPrimaryAccount,
+    makeTransferFromAdminPrimaryAccount
```
Comment (coeniebeyers) [inline lib/index.js]: @JasoonS There is a comma missing here (I had it missing in my example as well)

**Comment by coeniebeyers** [comment]
Note upgrading to v0.0.5* requires https://github.com/AdharaProjects/cbs-proxy/pull/23 to be merged. Before then, stay on v0.0.4

**Pull request merged**